### ISSUE TITLE
A list with just placeholders

### ISSUE THREAD

**Issue by jonathanphz** [a5hik/ng-sortable#235]
Is there a way to have two lists, one with data/text and the other one with just placeholders where you can drop the data/text list?

for example:

List A                       List B 
* item 1                     * Placeholder
* item 2                     * Placeholder
* item 3                     * Placeholder

Then drag items from List A to List B.

Thanks!

**Comment by a5hik** [comment]
yup, this is the default behavior, and you can also clone the items!

**Comment by jonathanphz** [comment]
Hi - Sorry what I mean is that  would like the placeholder list to actually display a few empty placeholder boxes where you drop from the other list.

Example:

![item-lists](https://cloud.githubusercontent.com/assets/1545577/10670475/0acc6340-78bc-11e5-85a8-8f66657be28b.jpg)

**Comment by a5hik** [comment]
@jonathanphz you can disable the drag and drop on the place holder list (or it need not be an sortable list). and replace the html once the item is dropped on the 'itemMoved' callback.
